package.json

```diff
@@ -13,7 +13,16 @@
     "node": "0.10.39",
     "npm": "2.14.1"
   },
-  "dependencies": {},
+  "dependencies": {
+    "babel": "^5.8.34",
+    "classnames": "^2.2.1",
```

Comment:
We need to use fixed versions rather than caret ranges, to ensure builds can be reproduced exactly and easily.

Additionally, most of the things you've listed here are dev dependencies rather than what we're currently calling "production". They also duplicate things in the dev dependencies section...